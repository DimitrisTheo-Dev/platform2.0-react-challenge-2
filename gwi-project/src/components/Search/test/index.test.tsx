import { render, screen } from "@testing-library/react"
import { MemoryRouter } from "react-router-dom"
import { Search } from "../index"

test("render search", () => {
  render(
    <MemoryRouter>
      <Search />
    </MemoryRouter>
  )
  const searchInput = screen.getByTestId("search-input")
  const search = screen.getByTestId("search")
  expect(search).toBeInTheDocument()
  expect(searchInput).toBeInTheDocument()
})

describe("Search", () => {
  it("should render", () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Search />
      </MemoryRouter>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
