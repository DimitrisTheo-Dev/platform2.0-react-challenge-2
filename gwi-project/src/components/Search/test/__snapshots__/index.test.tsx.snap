// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search should render 1`] = `
<DocumentFragment>
  <div
    class="css-4kutbl"
    data-testid="search"
  >
    <input
      data-testid="search-input"
      placeholder="Search"
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
